src/get-started/wordcount-example.md

```diff
@@ -677,15 +801,22 @@ $ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WindowedWordCo
      -Pdataflow-runner
 ```
 
-To view the full code in Java, see **[WindowedWordCount](https://github.com/apache/beam/blob/master/examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java).**
+To view the full code in Java, see
+**[WindowedWordCount](https://github.com/apache/beam/blob/master/examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java).**
 
 > **Note:** WindowedWordCount is not yet available for the Python SDK.
 
 ### Unbounded and bounded pipeline input modes
 
-Beam allows you to create a single pipeline that can handle both bounded and unbounded types of input. If the input is unbounded, then all PCollections of the pipeline will be unbounded as well. The same goes for bounded input. If your input has a fixed number of elements, it's considered a 'bounded' data set. If your input is continuously updating, then it's considered 'unbounded'.
+Beam allows you to create a single pipeline that can handle both bounded and
+unbounded types of input. If the input is unbounded, then all PCollections of
+the pipeline will be unbounded as well. The same goes for bounded input. If your
```

Comment:
all downstream PCollections will be unbounded, but separate branches may be independently bounded; running a pipeline which contains an unbounded PCollection requires streaming

(above is not site-suitable wording)

Reply:
changed wording